common/src/main/java/org/arpha/dto/media/enums/UploadType.java

```diff
@@ -0,0 +1,5 @@
+package org.arpha.dto.media.enums;
+
+public enum UploadType {
+    USER, PRODUCT
+}
```

Comment:
Please consider renaming this enum to something like TargetType or UploadTarget. As of now enum name does not represent its contents